src/Avalonia.Visuals/Rendering/DeferredRenderer.cs

```diff
@@ -31,7 +32,7 @@ public class DeferredRenderer : RendererBase, IRenderer, IVisualBrushRenderer
         private Scene _scene;
         private IRenderTarget _renderTarget;
         private DirtyVisuals _dirty;
-        private IRenderTargetBitmapImpl _overlay;
+        private IRef<IRenderTargetBitmapImpl> _overlay;
```

Comment:
Do all the bitmaps here and in `RenderLayer` etc need to be reference counted? They're owned by the renderer and won't be shared with the UI thread.

Reply:
We can't pass a raw `IBitmapImpl` to DrawingContext anymore. There is `RefCountable.CreateUnownedNotClonable<T>` method, but I'd like to avoid calling it when possible, since it provides an "incomplete" implementation of `IRef<T>`.